frontend/src/pages/TutorialPage/index.tsx

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import "./index.css";
+import { Helmet } from "react-helmet";
+import { useGoogleAnalytics } from "../../components/useGoogleAnalytics";
+import { Divider, Typography, Image } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
+
+const { Title, Paragraph, Text, Link } = Typography;
+/**
+ * Tutorial Page
+ */
+
+export function TutorialPage() {
+  useGoogleAnalytics();
+
+  return (
+    <div className="page">
+      <Helmet>
+        <title>ExplainaBoard - Tutorials</title>
+      </Helmet>
+      <Typography>
+        <Title>What is ExplainaBoard? Why Do We Need It?</Title>
+        <Paragraph>
+          The reason why we created ExplainaBoard is to do automatic evaluation
+          for users. We want to be able to save time after they already took a
+          long time training and tuning their models.
+        </Paragraph>
+
+        <Title level={2}>Getting Started</Title>
+        <Paragraph>
+          In this quickstart guide, we will walk you through the basic
+          functionalities of ExplainaBoard:
+        </Paragraph>
+        <Paragraph>
+          <ul>
+            <li>
+              <a href="#upload-system">Upload a system </a>
+            </li>
+            <li>
+              <a href="#view-analysis">
+                View detailed analysis of your outputs
+              </a>
+            </li>
+          </ul>
+        </Paragraph>
+        <Divider />
+        <Title level={2} id="upload-system">
+          Upload a System
+        </Title>
+        <Paragraph>
+          We will be using a text classification example for this tutorial.
+          Please download the <Text code>output_sst2.txt</Text> file{" "}
```

Comment:
It'd be really nice if we could give an example of pairwise analysis, as pairwise analysis is much more intuitive with respect to getting an idea of where one system is better than the other. We could use the CNN and LSTM output here: https://github.com/neulab/ExplainaBoard/tree/main/data/system_outputs/sst2